wallet/src/de/schildbach/wallet/service/BlockchainService.java

```diff
@@ -655,7 +654,7 @@ protected void onSuccess(final HostAndPort hostAndPort, final InetSocketAddress
                         log.info("trusted peer '{}' resolved to {}", hostAndPort,
                                 socketAddress.getAddress().getHostAddress());
                         if (socketAddress != null) {
-                            peerGroup.addAddress(new PeerAddress(Constants.NETWORK_PARAMETERS, socketAddress), 10);
+                            peerGroup.addAddress(PeerAddress.simple(socketAddress), 10);
```

Comment:
Check if this is the same as `new PeerAddress(Constants.NETWORK_PARAMETERS, socketAddress)`